ci-operator/step-registry/ipi/conf/additional-ca/trust-policy/ipi-conf-additional-ca-trust-policy-commands.sh

```diff
@@ -12,6 +12,7 @@ additional_trust_bundle="${SHARED_DIR}/additional_trust_bundle"
 
 if [[ $CLUSTER_TYPE == "nutanix" ]]; then
   if [[ -f "${CLUSTER_PROFILE_DIR}/prismcentral.pem" ]]; then
+    ADDITIONAL_TRUST_BUNDLE_POLICY="Always"
```

Comment:
Suggest to set `ADDITIONAL_TRUST_BUNDLE_POLICY` in `ipi-conf-nutanix` chain as a default Env.

Reply:
Since I added step - ref: ipi-conf-additional-ca-trust-policy to ipi-conf-nutanix-chain.yaml, for env like dev ci, it's not using   self-signed cert yet, so ADDITIONAL_TRUST_BUNDLE_POLICY should not be "Always"
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/66249/rehearse-66249-pull-ci-openshift-machine-api-operator-release-4.19-e2e-nutanix/1935604803581251584/artifacts/e2e-nutanix/ipi-install-install/artifacts/install-config.yaml

Reply:
On nutanix, only env on self-signed cert, ADDITIONAL_TRUST_BUNDLE_POLICY must be "Always"

Reply:
Can we set `ADDITIONAL_TRUST_BUNDLE_POLICY: Always` in ipi-conf-nutanix chain ?

Reply:
```
  env:
  - name: ADDITIONAL_TRUST_BUNDLE_POLICY
    default:  "Always"
```

Reply:
I don't think so, only env with self-signed cert, ADDITIONAL_TRUST_BUNDLE_POLICY must be "Always", dev's ci is not using self-signed cert yet

Reply:
@gpei `additionalTrustBundle` has dependency on `additionalTrustBundlePolicy`? Or they are two separate configurations.

Reply:
the same as ASH which also requiring the additional CA, https://steps.ci.openshift.org/reference/ipi-conf-additional-ca-azurestack , how about adding a dedicated step for Nutanix?

Reply:
so we can use `ipi-conf-additional-ca-trust-policy` as the original usage to test `additionalTrustBundle` & `additionalTrustBundlePolicy` options only, but not used for setting various parameters for different platforms

Reply:
if so in the new ipi-conf-additional-ca-nutanix step, we can easily patch `additionalTrustBundle` & `additionalTrustBundlePolicy` for Nutanix clusters that's using self-signed CA, and do nothing for Nutanix cluster not using self-signed cert

Reply:
agree.